lms/templates/registration/failed_password_reset_email.html

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}{% autoescape off %}
+{% blocktrans %}You're receiving this e-mail because you requested a password reset for your user account at {{ platform_name }}.{% endblocktrans %}
+
+{% blocktrans %}However, there is currently no user account associated with your email address: {{ email_address }}{% endblocktrans %}
+
+{% trans "If you didn't request this change, you can disregard this email." %}
+
+{% blocktrans %}The {{ platform_name }} Team{% endblocktrans %}
+
+{% endautoescape %}
```

Comment:
I feel like it should be possible to do this in 1 template, namely the `password_reset_email.html` template, by just passing in a "failed" boolean to allow us to branch to the right message body. What do you think?